src/hotspot/share/prims/jvmtiDeferredUpdates.hpp

```diff
@@ -26,10 +26,13 @@
 #ifndef SHARE_PRIMS_JVMTIDEFERREDUPDATES_HPP
 #define SHARE_PRIMS_JVMTIDEFERREDUPDATES_HPP
 
-#include "runtime/thread.inline.hpp"
+#include "runtime/javaThread.hpp"
 #include "utilities/globalDefinitions.hpp"
 #include "utilities/growableArray.hpp"
 
+class MonitorInfo;
+template <typename T> class GrowableArray;
+
 class jvmtiDeferredLocalVariable : public CHeapObj<mtCompiler> {
```

Comment:
This surprised me, but I'm guessing we picked up the template
from thread.inline.hpp...

Reply:
yes.